pallets/medical-record/src/tests.rs

```diff
@@ -59,3 +46,11 @@ fn patient_can_add_record() {
 		);
 	});
 }
+
+fn origin_to_account_id(o: RuntimeOrigin) -> u64 {
+	let ro: Result<RawOrigin<u64>, RuntimeOrigin> = o.clone().into();
+	match ro.ok().unwrap() {
```

Comment:
[Nitpicking]
Could you please change the `unwrap()` to a `expect()` just for the sake of clarity?